week1/homework/issues/github.md

```diff
@@ -0,0 +1,17 @@
+## Learning GitHub
+
+*For this assignment, you will follow the instructions and complete a task showing your knowledge of the subject at the end. If at any moment you need help, feel free to contact your TAs.*
+
+### GitHub
+
+If you haven't heard of GitHub before, you must have been living under a rock for the past few years. GitHub is a cloud-based repository hosting service that is widely used in the tech industry. It allows teams to use Git for version control, collaboration, and file management. If you don't know what Git is, it's a version-control system for tracking changes, managing state, and concurrently developing on the same files or directories. Git and Github's tools are specifically designed to make coordinating work easier, and they are one of the most pervasive shared tools among students and the industry.
```

Comment:
> If you haven't heard of GitHub before, you must have been living under a rock for the past few years.

If anyone hasn't heard of GitHub before, they're going to feel really bad reading this. Consider removing or rewording to make this more inclusive.